platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/ReplyToCommentTestIT.java

```diff
@@ -0,0 +1,1293 @@
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Created by dmardassi on 16/10/2017.
+ */
+@Tag("social")
+@Tag("sniff")
+public class ReplyToCommentTestIT extends Base {
+
+    ForumTopicManagement forumTopicManagement;
+
+    TasksManagement tasksManagement;
+
+    ForumForumManagement forumForumManagement;
+
+    ForumCategoryManagement forumCategoryManagement;
+
+    NavigationToolbar navigationToolbar;
+
+    AddUsers addUsers;
+
+    ManageLogInOut manageLogInOut;
+
+    HomePagePlatform homePagePlatform;
+
+    ConnectionsManagement connectionsManagement;
+
+    ActivityStream activityStream;
+
+    ForumHomePage forumHomePage;
+
+    ProjectsManagement projectsManagement;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+        forumHomePage = new ForumHomePage(this);
+        forumForumManagement = new ForumForumManagement(this);
+        forumCategoryManagement = new ForumCategoryManagement(this);
+        navigationToolbar = new NavigationToolbar(this);
+        forumTopicManagement = new ForumTopicManagement(this);
+        homePagePlatform = new HomePagePlatform(this);
+        addUsers = new AddUsers(this);
+        manageLogInOut = new ManageLogInOut(this);
+        connectionsManagement = new ConnectionsManagement(this);
+        activityStream = new ActivityStream(this);
+        projectsManagement = new ProjectsManagement(this);
+        tasksManagement = new TasksManagement(this);
+        if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+            $(ELEMENT_SKIP_BUTTON).click();
+        }
+        if ($(ELEMENT_INPUT_USERNAME_CAS).is(Condition.not(Condition.exist))) {
+            manageLogInOut.signOut();
+        }
+        manageLogInOut.signInCas(PLFData.DATA_USER1, "gtngtn");
+    }
+
+    @Test
+    public void test01_ReplyToCommentInAS() {
+        String activity1 = "activity1" + getRandomNumber();
+        String comment = "comment" + getRandomNumber();
+        String reply = "Reply" + getRandomNumber();
+        homePagePlatform.goToConnections();
+        connectionsManagement.connectToAUser(DATA_USER2);
+        homePagePlatform.goToHomePage();
+        activityStream.addActivity(activity1, "");
+        manageLogInOut.signIn(DATA_USER2, PLFData.DATA_PASS);
+        homePagePlatform.goToConnections();
+        connectionsManagement.acceptAConnection(DATA_USER1);
+        homePagePlatform.goToHomePage();
+        activityStream.commentActivity(activity1, comment);
+        manageLogInOut.signOut();
+        manageLogInOut.signInCas(PLFData.DATA_USER1, "gtngtn");
+        homePagePlatform.goToHomePage();
+        activityStream.replytocomment(comment, reply, DATA_NAME_USER1);
+        refresh();
```

Comment:
Shouldn't we check if the reply exists and has the right content ?

Reply:
I checked it in the method replytocomment()